src/main/java/com/juniorstart/juniorstart/model/UserProfile.java

```diff
@@ -0,0 +1,43 @@
+package com.juniorstart.juniorstart.model;
+
+import lombok.*;
+import javax.persistence.Entity;
+import javax.persistence.*;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.UUID;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@EqualsAndHashCode(exclude="userTechnology")
+@Entity
+@Table(name="users_profile")
+@Builder
+public class UserProfile {
+    @Id
+    private UUID privateId;
+
+    @OneToOne
+    @MapsId
+    private User user;
+
+    @Enumerated(EnumType.STRING)
+    private UserRole userRole;
+
+    @ManyToMany(cascade = CascadeType.PERSIST)
+    @JoinTable(name="join_user_technology",
+    joinColumns = {@JoinColumn(name="user_id")},
+    inverseJoinColumns = {@JoinColumn(name="technology_id")})
+    private Set<UserTechnology> userTechnology = new HashSet<>();
+
+    public void addUserTechnology(UserTechnology technology){
```

Comment:
Czemu są tu te metody, przecież adnotacja @Data sama je wygeneruje.

Reply:
Faktycznie zmieniłem.

Reply:
Czemu addUserTechnology jest zakomentowane? Do czego to?

Reply:
Przez przypadek zamiast usunąć dodałem komentarz, już usunąłem.